ed25519_utils.go

```diff
@@ -0,0 +1,29 @@
+package jwt
+
+import (
+	"encoding/pem"
+	"golang.org/x/crypto/ed25519"
+)
+
+
+// Parse PEM encoded Elliptic Curve Private Key Structure
+func ParseED25519PrivateKeyFromPEM(key []byte) (*ed25519.PrivateKey, error) {
```

Comment:
`Ed` is an abbreviation, not an acronym. Don't include the curve name, for symmetry with `ParseECPrivateKeyFromPEM`.

`ParseEdPrivateKeyFromPEM`

Reply:
Seems reasonable.